localization/hi/adapter/README.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: pattern
+title: Adapter
+folder: adapter
+permalink: /patterns/adapter/hi
+categories: Structural
+language: hi
+tags:
+- Gang of Four
+---
```

Comment:
Have a look at the updated yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute. You can drop some of the tags. Check this for all the patterns in this PR.